quantum/quantum.c

```diff
@@ -322,50 +322,101 @@ bool process_record_quantum(keyrecord_t *record) {
-                                                                         // (     )        *        +        ,        -        .        /
-                                                                         KC_9, KC_0, KC_8, KC_EQL, KC_COMM, KC_MINS, KC_DOT, KC_SLSH,
-                                                                         // 0     1        2        3        4        5        6        7
-                                                                         KC_0, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7,
-                                                                         // 8     9        :        ;        <        =        >        ?
-                                                                         KC_8, KC_9, KC_SCLN, KC_SCLN, KC_COMM, KC_EQL, KC_DOT, KC_SLSH,
-                                                                         // @     A        B        C        D        E        F        G
-                                                                         KC_2, KC_A, KC_B, KC_C, KC_D, KC_E, KC_F, KC_G,
-                                                                         // H     I        J        K        L        M        N        O
-                                                                         KC_H, KC_I, KC_J, KC_K, KC_L, KC_M, KC_N, KC_O,
-                                                                         // P     Q        R        S        T        U        V        W
-                                                                         KC_P, KC_Q, KC_R, KC_S, KC_T, KC_U, KC_V, KC_W,
-                                                                         // X     Y        Z        [        \        ]        ^        _
-                                                                         KC_X, KC_Y, KC_Z, KC_LBRC, KC_BSLS, KC_RBRC, KC_6, KC_MINS,
-                                                                         // `     a        b        c        d        e        f        g
-                                                                         KC_GRV, KC_A, KC_B, KC_C, KC_D, KC_E, KC_F, KC_G,
-                                                                         // h     i        j        k        l        m        n        o
-                                                                         KC_H, KC_I, KC_J, KC_K, KC_L, KC_M, KC_N, KC_O,
-                                                                         // p     q        r        s        t        u        v        w
-                                                                         KC_P, KC_Q, KC_R, KC_S, KC_T, KC_U, KC_V, KC_W,
-                                                                         // x     y        z        {        |        }        ~        DEL
-                                                                         KC_X, KC_Y, KC_Z, KC_LBRC, KC_BSLS, KC_RBRC, KC_GRV, KC_DEL};
+#ifndef USE_PACKED_KEYCODE_LUT
```

Comment:
If it's demonstrably better I think I'd prefer just having the single implementation -- flash size comes at a premium in comparison to a few bits of arithmetic, so I think I'd prefer to just use the packed versions under normal circumstances anyway.

Reply:
As far as I can tell, the new code performs as good as the old, or at least made no appreciable difference.

So we're down to my remaining question of "does just using the packed version constitute a breaking change".